airbyte-integrations/bases/base-java/build.gradle

```
@@ -5,6 +5,12 @@ plugins {
 
 dependencies {
     implementation 'commons-cli:commons-cli:1.4'
+    implementation group: 'org.apache.sshd', name: 'sshd-mina', version: '2.7.0'
```

```suggestion
    implementation 'org.apache.sshd:sshd-mina:2.7.0'
```
nit: we generally use this format